### ISSUE TITLE
Include test data files

### ISSUE THREAD

**Issue by karawoo** [Sage-Bionetworks/dccvalidator#311]
Based on feedback from users, it would be helpful to make available the test files we use for demos. I'm not sure which would be the most ideal way to do this: we could put them in the Synapse project where the metadata templates are stored, or bundle them with the app in `inst/extdata`. 

Should we add a button to the app that downloads the files from wherever they're stored to the user's machine? If so, where's the best place for that? Could go in the instructions, but it might be easily overlooked there. Thoughts, @Aryllen and @kelshmo?

**Comment by kelshmo** [comment]
I don't think we have the best strategy implemented - I think it is easily overlooked. Currently, a PEC user could find them in [this Wiki](https://www.synapse.org/#!Synapse:syn2787333/wiki/599260), which is private to that group. 

I don't see an obvious place for a button within the app and I think this could best live within Synapse anyway. I will move the example files, and add a more obvious link, to [where the templates are stored](https://www.synapse.org/#!Synapse:syn20729790).

**Comment by cw-dvr8** [comment]
@kelshmo How about waiting until after the demo tomorrow to move the files?  I've updated all of the ones that I'm planning on using.

**Comment by Aryllen** [comment]
I'm a bit torn on this. I think a "Download Test Templates" button might be able to go next to the "Show Instructions" button, but it might also confuse people. Alternatively, it could be in the "Using the App" tab under a section called something like "Try Out the App". It wouldn't necessarily need to be a button, but a link to a Synapse folder that holds the test templates.

**Comment by karawoo** [comment]
Just an update that we have test files for AMP-AD here: https://www.synapse.org/#!Synapse:syn21781041

**Comment by Aryllen** [comment]
One challenge with demo files is that they become out of date as we update templates. It would be handy if the demos would always be up to date with the latest version of the templates, even if that means new key columns have no values.
